pkgs/tools/networking/chisel/default.nix

```diff
@@ -17,6 +17,8 @@ buildGoModule rec {
 
   vendorSha256 = "01wh8fn76jn8hnf7gj759k8dwqrr0p36xmsxfc8dayzinpl10fqv";
 
+  buildFlagsArray = [ "-ldflags=-X github.com/jpillora/chisel/share.BuildVersion=${version}" ];
```

Comment:
```suggestion
  buildFlagsArray = [ "-ldflags=-s -w -X github.com/jpillora/chisel/share.BuildVersion=${version}" ];
```

Reply:
It feels like that `-s -w` should be applied by default but that's more a personal opinion.